packages/compile-solidity/src/compilerSupplier/index.ts

```diff
@@ -67,6 +70,11 @@ export class CompilerSupplier {
 
     if (strategy) {
       const solc = await strategy.load(userSpecification);
+      debug(
+        "Loaded solc v%s using %s strategy",
+        solc.version(),
```

Comment:
oh, this is convenient!

Reply:
Yeah but I suspect it's causing test timeouts? On Docker, this will shell out (maybe that's something to cache)

Reply:
Wait I was wrong; it doesn't shell out. Huh I wonder why the addition of some debug statements would cause worse timeouts. Maybe it's environmental? I'll try again in the morning.

Reply:
(Reverting the two commits to add instrumentation to test that those are what's causing the timeouts)

Reply:
I thought it was convenient that all loaded strategies would return a `version` function.  bummer.